src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/InputInnerAssignment.java

```diff
@@ -19,7 +19,8 @@ void innerAssignments()
         int b;
         int c;
 
-        a = b = c = 1; // violation
+        a = b = // violation
+                c = 1; // violation
```

Comment:
@shashwatj07 , @timurt 

can we keep single line in input ?

as violation is same can we have `a = b = c = 1;  // 2 violations`

we do not need to go crazy with parsing, we can simply support matching by 

`// violation`
`// 2 violations`

in future:
it is will be very strange  code that will require more than 2 violation. even we come to it, we can extend support to
`// 3 violation` .... `// 9 violation` . Above 9 .... I want to see real example, it should not make any sense.

Reply:
We can definitely do that. What about violation messages?

Reply:
For now we will support only the same message for both violations

Reply:
Okay

Reply:
https://github.com/checkstyle/checkstyle/issues/10190

Reply:
fixed